deployment/provision_vagrant_vm.sh

```diff
@@ -44,7 +44,8 @@ cp /etc/skel/.bashrc /home/$USER/
 OWNER=$(stat -c %u "$MOUNTPOINT/evap")
 apt-get -q install -y bindfs
 mkdir -p "$REPO_FOLDER"
-bindfs --map="$OWNER/1042:@$OWNER/@1042" "$MOUNTPOINT" "$REPO_FOLDER" || exit 1
+# remount if REPO_FOLDER does not contain any files (meaning it is not mounted)
+[[ -z $(ls -A "$REPO_FOLDER") ]] && bindfs --map="$OWNER/1042:@$OWNER/@1042" "$MOUNTPOINT" "$REPO_FOLDER" || exit 1
```

Comment:
change to if-else clause with a logging message in the else clause

Reply:
@Kakadus done